rocky/reports/views/generate_report.py

```diff
@@ -234,23 +239,70 @@ class SetupScanGenerateReportView(
     current_step = 3
 
     def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        del request.session["report_name"]  # delete session
+        del request.session["reference_date"]  # delete session
```

Comment:
The comments aren't necessary imo, this should be common (Django) knowledge. Also, you're not deleting a session, but a session key/ record